hazelcast/src/main/java/com/hazelcast/internal/usercodedeployment/impl/ClassData.java

```diff
@@ -31,18 +33,18 @@
  */
 public class ClassData implements IdentifiedDataSerializable {
```

Comment:
I think `ClassData` name is misleading with this change. It should be something like `ClassloaderData`.